packages/tasks/src/model-libraries.ts

```diff
@@ -1370,6 +1370,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.vibevoice,
 		filter: false,
 	},
+	"videox_fun": {
+		prettyLabel: "VideoX Fun",
+		repoName: "VideoX Fun",
+		repoUrl: "https://github.com/aigc-apps/VideoX-Fun",
+		filter: false,
+		countDownloads: `path_extension:"safetensors"`,
```

Comment:
This rule means that downloading any safetensors files from the repo will count as a download. This can be detrimental if multiple safetensors files are present in the repo as they would double-count them.

Looking at https://huggingface.co/alibaba-pai/CogVideoX-Fun-V1.5-5b-InP, it seems there are a few json config files in the repo. Would it be possible to count downloads on a specific one so we are sure we never double count?